src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -4783,6 +4788,20 @@ private void VisitMemberAccess(BoundExpression node, BoundExpression receiverOpt
             }
 
             int getReceiverSlot() => (receiverOpt is null) ? -1 : MakeSlot(receiverOpt);
+
+            int getMemberSlot()
```

Comment:
Seems like the same logic as in the change to `LocaldataFlowPass`. Should we add a helper both can use?

Reply:
Unfortunately the two implementations differ slightly with respect to `receiver` and `member` so it may be simpler to keep distinct implementations.

---
In reply to: [253697166](https://github.com/dotnet/roslyn/pull/33121#discussion_r253697166) [](ancestors = 253697166)